src/webui/btjson.cpp

```diff
@@ -708,6 +711,7 @@ QVariantMap getTranserInfoMap()
     map[KEY_TRANSFER_AVERAGE_TIME_QUEUE] = cacheStatus.averageJobTime;
     map[KEY_TRANSFER_TOTAL_QUEUED_SIZE] = cacheStatus.queuedBytes;
 
+    map[KEY_TRANSFER_FREESPACEONDISK] = Utils::Fs::freeDiskSpaceOnPath(BitTorrent::Session::instance()->defaultSavePath());
```

Comment:
You can see an example of QtConcurrent::run() being used in [speedwidget.cpp](https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/properties/speedwidget.cpp#L117).